### PR TITLE
snake case for package's name

### PR DESCRIPTION
**Pull request by AndreaScorza**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = lto-cli
-version = 0.0.10
+version = 0.0.11
```
Comment (KaloyanTanev) [inline setup.cfg:3]: Do you update the version on every PR?

File: tests/AccountTest.py [hunk 2]
```diff
@@ -1,37 +0,0 @@
-from LTOCli.Commands import account
```
Comment (KaloyanTanev) [inline tests/AccountTest.py:1]: Why no tests anymore? (goes for other deleted files as well)

File: setup.cfg (hunk 1 above)
Comment (AndreaScorza) [reply setup.cfg:3]: same on the other PR, you have to, not my choice

File: tests/AccountTest.py (hunk 2 above)
Comment (AndreaScorza) [reply tests/AccountTest.py:1]: those tests were useless. Need to create new ones, and they should be e2e.

Comment (AndreaScorza) [reply tests/AccountTest.py:1]: there are no other deleted files except the tests

Comment (KaloyanTanev) [reply tests/AccountTest.py:1]: Okay.

**Review by KaloyanTanev — APPROVED** [review]

**Pull request merged**